src/mlpack/core/tree/cover_tree/traits.hpp

```diff
@@ -33,6 +33,12 @@ class TreeTraits<CoverTree<MetricType, StatisticType, MatType, RootPointPolicy>>
   static const bool HasOverlappingChildren = true;
 
   /**
+   * Cover trees do have self-children, so points can be included in more than
+   * one node.
+   */
+  static const bool HasDuplicatedPoints = true;
```

Comment:
A classic example of me talking before sufficiently analyzing what I am looking at.  Let's leave it as-is. :)